n1netails-liquibase/src/main/resources/db/changelog/sql/db.changelog-00012-create-notification-template-table.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE IF NOT EXISTS ntail.email_notification_template
+(
+    id character varying(100),
+    subject TEXT,
+    html_body TEXT,
+    CONSTRAINT email_notification_template_pkey PRIMARY KEY (id)
+);
+
+ALTER TABLE IF EXISTS ntail.email_notification_template
+    OWNER to n1netails;
```

Comment:
make sure to create a sequence number generator for the id value something like

`CREATE SEQUENCE IF NOT EXISTS email_seq START WITH 1 INCREMENT BY 1;`